tests/bitcoin/BitcoinProcessor.spec.ts

```diff
@@ -23,7 +23,9 @@ function randomBlock (above: number = 0): IBlockInfo {
 describe('BitcoinProcessor', () => {
 
   const testConfig: IBitcoinConfig = {
-    bitcoinPeerUri: 'http://localhost:18331',
+    bitcoinPeerUri: 'http://localhost:18332',
+    bitcoinRpcUsername: 'admin',

```

Comment:
Need @thehenrytsai since this reaches into overarching config details, but theres no strict requirement on location and the port will change between testnet and mainnet. For the time being I've set the src/bitcoin-config to a local testnet node.